daml-foundations/daml-ghc/package-database/util.bzl

```
@@ -155,20 +155,16 @@ def _daml_package_db_impl(ctx):
             """
         set -eou pipefail
         shopt -s nullglob
-        mkdir -p {db_dir}
-        for ver in {daml_lf_versions}; do
-            mkdir -p "{db_dir}/$ver/package.conf.d"
-        done
-        """.format(db_dir = db_dir.path, daml_lf_versions = " ".join(ctx.attr.daml_lf_versions)) +
+        mkdir -p "{db_dir}/package.conf.d"
+        """.format(db_dir = db_dir.path, daml_lf_version = ctx.attr.daml_lf_version) +
```

`daml_lf_version` looks unused in this `.format()`. 
```suggestion
        """.format(db_dir = db_dir.path) +
```